plugins/jdg72/pom.xml

```diff
@@ -15,6 +15,9 @@
 
    <properties>
       <version.infinispan>${jdg72.infinispan.version}</version.infinispan>
+      <version.infinispan.public>8.2.1.Final</version.infinispan.public>
+      <version.narayanajta>5.2.17.Final-redhat-1</version.narayanajta>
+      <version.jta-spec>1.0.1.Final-redhat-2</version.jta-spec>
```

Comment:
I removed this in https://github.com/radargun/radargun/pull/518, and you can remove this file from the PR